### PR TITLE
Update prettier to v2 and install husky and pretty-quick to perform p…

### PR DESCRIPTION
**Pull request by emmalewis**
…rettier on a pre-commit hook

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
   "printWidth": 100,
-  "singleQuote": true,
-  "trailingComma": "es5"
```
Comment (emmalewis) [inline .prettierrc:4]: (trailingComma is es5 by default in v2 of prettier)

Comment (aendrew) [reply .prettierrc:4]: Honestly I think you can just remove this whole file, eslint config prettier will align with whatever the lint preference is

**Review by aendrew — CHANGES_REQUESTED** [review]
Please remove `.prettierrc`; I think all those settings are handled by Airbnb.

**Comment by emmalewis** [comment]
@aendrew The .prettierrc is still necessary to enforce the singleQuotes and printWidth with local development (formatOnSave etc).

**Review by aendrew — APPROVED** [review]
Ah, right you are! You DO need Prettier config! Apologies!

https://github.com/prettier/eslint-config-prettier#quotes

**Pull request merged**